pom.xml

```diff
@@ -196,6 +196,9 @@
 				<configuration>
 					<source>17</source>
 					<target>17</target>
+					<compilerArgs>
```

Comment:
I'd like to leave this on. I sometimes see a deprecation and don't really know where it comes from